{{cookiecutter.project_slug}}/src/features/{% if cookiecutter.has_maps_blueprint == "True" %}MapsScreen{% endif %}/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import { StyleSheet, Image, View, Alert, } from "react-native";
+import { Text, Layout, Button } from "react-native-ui-kitten";
+import LogoIcon from "../../assets/images/backgroundLoginV1.png";
+import { installed_blueprints } from "../../config/installed_blueprints";
+import { store } from "../../redux/store";
+import MapView, { PROVIDER_GOOGLE } from 'react-native-maps';
+import { SearchBar } from 'react-native-elements';
+
+
+export default class MapsScreen extends Component {
+  componentDidMount() {
+    store.dispatch({ type: "TEST/ALO" });
+  }
+
+  state = {
+    search: '',
+  };
+
+  updateSearch = search => {
+    this.setState({ search });
+  };
+
+  render() {
+    const { search } = this.state;
+
+    return [
+      <MapView
+        provider={PROVIDER_GOOGLE} // remove if not using Google Maps
+        style={styles.map}
+        region={{
+          latitude: 37.78825,
+          longitude: -122.4324,
+          latitudeDelta: 0.015,
+          longitudeDelta: 0.0121,
+        }}
+      >
+      </MapView>,
+
+      <SearchBar
```

Comment:
We already have a UI library, React Native UI Kitten, so please re do this component using that in mind.

https://akveo.github.io/react-native-ui-kitten/docs/getting-started/

I am checking that they don't have a search bar, so you will have to create one using `Input` and a search icon.

If you really want to do it perfect, you can create this SearchBar component in the `src/components` folder where all other scaffolds will be able to use your SearchBar 👍